python/tvm/relay/frontend/pytorch.py

```diff
@@ -2120,14 +2146,17 @@ def _get_constant(node):
         attr_name = attribute_names[0]
         ty = node.output().type().kind()
 
-        if ty in ["IntType", "BoolType"]:
+        if ty == "IntType":
```

Comment:
why change this?

Reply:
I really wanted to cast bool(...) for bool types. Somehow that didn't survive the refactors of the PR. Do you want me to merge the two or add the cast?

Reply:
ok, makes sense